numpy/core/tests/test_multiarray.py

```diff
@@ -2907,7 +2907,8 @@ def norm(result):
                                      "__array_ufunc__")
                     else:
                         if (isinstance(obj, np.ndarray) and
-                                not hasattr(obj, "__array_ufunc__")):
+                            (obj.__class__.__array_ufunc__ is
```

Comment:
Could that be `obj.__array_ufunc__` instead? It shouldn't be in the instance dictionary unless the `__init__` function puts it there, in which case it is an override.